### ISSUE TITLE
Stuck at "Connected" while following the procedure of patching the client.

### ISSUE THREAD

**Issue by RealNazGul** [The-Cataclysm-Preservation-Project/TrinityCore#329]
Hello, I realize that this has been reported numerous times in the issue tracker and multitude of help comments exist, but I believe I am following the procedure of patching the client correctly and yet I am unable to login.

**Description:**

I am unable to login to a core running at a VPS based on Ubuntu 21.10 with version "TrinityCore rev. 8a5e6f79e243+ 2022-02-02 12:26:08 +0100 (master branch) (Unix, debug, Static) (bnetserver)".

**Current behaviour:**

The client gets stuck for a few minutes at "Connecting" and then opts into "Unable to connect" message.

**Expected behaviour:**

After registering a battlenet account inside the worldserver console and then typing the battle.net account's email address and it's password into a patched 4.3.4 client (using the client downloader provided by this project) being able to get to realms listing and possibly connecting to a realm, making a character and getting in-game!

**Steps to reproduce the problem:**

1. I downloaded the source code of the core & DB, then compiled at Linux Ubuntu 21.10 using TrinityCore's wiki pages for reference on the additional programs needed.
2. Then I got the client downloader from: https://github.com/The-Cataclysm-Preservation-Project/ClientDownloader/releases/tag/1.1
3. I also download the full client with: https://github.com/The-Cataclysm-Preservation-Project/wowregeneration
4. I've ran the server with the cameras, dbc, maps, vmaps, mmaps and set the realmlist at the server's IP.
5. I've patched the client using natively compiled Linux binary with the following command "./connection_patcher Wow.exe", same for 64 bit client. Here I got the error about auth modules not being able to be downloaded, then I downloaded the auth modules at: https://github.com/The-Cataclysm-Preservation-Project/AuthModules - patcher log said successfully patched.
6. I've set the patchlist to 127.0.0.1 and the portal to the server's IP.
7. Disabled firewall on the host machine, just to be sure it won't interfere, although ports were forwarded - no router exists.
8. Upon connecting I see a bunch of things as:
- Patcher log: https://gist.github.com/RealNazGul/dd333b8bfca67eaae77a64b47b2494b0
- Authserver log file: https://gist.github.com/RealNazGul/adc6dc5f52352cc3f1ecfa39a5108bd6
- Client connection.log file: https://gist.github.com/RealNazGul/a912117df4c80913055b39ec221f03e1

I believe I have trouble with the .auth files as seen in the logs, however I tried to repeat the process and got the exact same result, do note that server, patcher and client data extractors are native Linux programs, but the Wow client is a windows binary (.exe/.dll).
I am going to try now patching the client under Windows to see if it will have any effect.

**TC rev. hash/commit:** 
TrinityCore rev. 8a5e6f79e243+ 2022-02-02 12:26:08 +0100 (master branch) (Unix, debug, Static) (bnetserver)
TrinityCore rev. 8a5e6f79e243+ 2022-02-02 12:26:08 +0100 (master branch) (Unix, debug, Static) (worldserver-daemon)

**TDB version:**
TDB 434.22011

**Operating system:** 
Server: Ubuntu 21.10
Client: Ubuntu 21.10
Note: I am trying to connect from a different machine over the internet than where the server runs at.

**Comment by Ovahlord** [comment]
did you delete the launcher.exe?

**Comment by RealNazGul** [comment]
Yes, here's how the client looks and the "Wow." exes and battle net dlls are the patched ones.
https://i.imgur.com/3Njssez.png

**Comment by Ovahlord** [comment]
Since you are obviously using an emulator on linux, make sure the pathing for the auth module is properly handled because the client knows no mercy with that. You need the programm data part at all cost

**Comment by RealNazGul** [comment]
I see, here's how the ProgramData directory looks as: https://i.imgur.com/QCJBzkG.png
The 2 modules were downloaded manually by me and set into their correct folders as seen in other issues of such kind, the rest was created by running the client - not sure if that's okay or not.

* On the first run of the patcher it gave an error that it cannot download the auth modules, so I downloaded them and put them into the directories as seen above, I presume that's the correct path they should have?
* launcher.exe is still within the directory but in a sub-folder I created as seen in the screenshot of my previous comment, should they be in a completely different directory?
* Hmm, ports are all free, checked with telnet and ping. 

Thank you for the messages! Really appreciate your fast responses. :)
